internal/querycoordv2/task/utils.go

```diff
@@ -185,6 +185,10 @@ func fillSubChannelRequest(
 	req *querypb.WatchDmChannelsRequest,
 	broker meta.Broker,
 ) error {
+	indexes, err := broker.DescribeIndex(ctx, req.GetCollectionID())
+	if err == nil {
+		req.IndexInfoList = indexes
+	}
```

Comment:
err != nil doesn't need to be handled?

Reply:
Thanks. I added a log here. If pulling index meta failed, growing index would not be created. User may not turn on the index function, so we just ignore it if pulling failed.